src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -123,6 +123,9 @@
   <data name="SideBarOpenInNewWindow.Text" xml:space="preserve">
     <value>Open in new window</value>
   </data>
+  <data name="SideBarRestoreRecycleBin.Text" xml:space="preserve">
```

Comment:
```suggestion
  <data name="RestoreAllItems" xml:space="preserve">
```

Reply:
This key is already used in for the recycle bin inner toolbar. 
The problem is that all the words are capitalized, and we don't want that capitalization in the flyout menu. 
I could make a `FirstCharToUpper()` method to use it, but I'm not sure how it would react for non-ascii characters (like Chinese).

Reply:
How about SidebarRestoreAllItems? The `.Text` isn't needed.

Reply:
Alright, will fix that!